app/Models/Game.php

```diff
@@ -170,6 +170,15 @@ public static function boot()
                     $originalTitle,
                     $freshGame->title
                 );
+
+                // Update the canonical title in game_titles.
+                if (!$game->wasRecentlyCreated) {
+                    $canonicalTitle = $freshGame->titles()->where('is_canonical', true)->first();
+                    if ($canonicalTitle) {
+                        $canonicalTitle->title = $freshGame->title;
+                        $canonicalTitle->save();
+                    }
+                }
```

Comment:
Event name changes seem to be getting synced to game_titles, but hub name changes are not.

Reply:
May need help understand this. Developers shouldn't be touching hub games anymore, and they aren't generated anymore when creating new hubs. They only exist to back redirects to modern "Hubs 2.0".

Alternative titles for hubs is not in scope for this PR, and I'm not sure if it should be generally supported.

Reply:
I'm not saying they _should_ be supported. I'm just letting you know that they exist (because there's a backing GameData record). 

After digging into a bit more - the event Title is pulled from (and updated to) the `GameData` record, so it's getting sync'd to the `game_titles` row. When editing a hub Title, the `game_set.title` is updated, but not the `GameData.Title` 

So it appears the behavior I've noticed is the result of a "detached" `GameData` record for the hub.